tools/ci_build/github/azure-pipelines/templates/use-android-emulator.yml

```diff
@@ -13,7 +13,21 @@ parameters:
   type: boolean
   default: false
 
+- name: install_missing_packages
+  type: boolean
+  default: false
+
 steps:
+- ${{ if eq(parameters.install_missing_packages, true) }}:
+  - script: |
+      set -e -x
+      ${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager --install "emulator"
+      ${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager --install "platform-tools"
+      ${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager --list
+      sudo apt install -y adb
```

Comment:
is adb already included in platform-tools?

Reply:
Good question. We should not install it again.

Reply:
It should but not found it.

Reply:
The one you installed from apt-get probably doesn't match the Android NDK version we use here. I guess the file is on disk, just it is not in PATH.

Reply:
Have you tried this get_sdk_tool_paths function?
https://github.com/microsoft/onnxruntime/blob/main/tools/python/util/android/android.py#L23